packages/webapp/src/containers/Log/HarvestAllocation/index.js

```diff
@@ -40,18 +46,29 @@ class HarvestAllocation extends Component {
       }
     });
     let sum = Object.keys(val).reduce((sum, key) => sum + Number(val[key]), 0);
+    console.log(sum);
 
-    if (sum !== Number(this.props.formData.quantity_kg)) {
-      toastr.error('Total does not equal the amount to allocate');
-    } else {
+    if (
+      sum >= Number(this.props.formData.quantity_kg) - 0.1 ||
+      sum < Number(this.props.formData.quantity_kg) + 0.1
+    ) {
```

Comment:
I think during allocation we should make the values add up on the dot rather than +- 0.01.
Just to take an example, the system would allow you to distribute your 0.01 harvest to multiple distributions of 0. A discrepancy of 0.01 is okay after a conversion but probably shouldn't be allowed when manually allocating the harvest amount.

Reply:
Also make this an and instead of an or